cobspec/src/main/java/com/saralein/cobspec/logger/AppLogger.java

```diff
@@ -0,0 +1,52 @@
+package com.saralein.cobspec.logger;
+
+import com.saralein.server.logger.Logger;
+import com.saralein.server.request.Request;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.time.Instant;
+
+public class AppLogger implements Logger {
```

Comment:
If we also have an `Application` class, this name would be inconsistent. I might suggest we either use `App` or `Application` everywhere.